.github/workflows/push.yml

```diff
@@ -68,7 +66,18 @@ jobs:
           push: true
           tags: ghcr.io/${{ github.repository }}:${{ needs.branch_meta.outputs.sha }}
           labels: ${{ steps.docker_meta_img.outputs.labels }}
-
+      - name: Build and push ${{ github.repository }} (file storage)
```

Comment:
Please over write the label `org.opencontainers.image.title` with an title who hat file-storage in it.
There for you need an own step with the metadata part